<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Otter::EmailAccountPreferencesDialogWindow</class>
 <widget class="QDialog" name="Otter::EmailAccountPreferencesDialogWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>832</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="10" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>SMTP password:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLineEdit" name="smtpPasswordLineEdit">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>IMAP password:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Your name:</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLineEdit" name="smtpServerUrlLineEdit"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="imapPasswordLineEdit">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QSpinBox" name="smtpServerPortSpinbox">
         <property name="maximum">
          <number>65535</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>IMAP user name:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Email address:</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>SMTP host:</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>SMTP port:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="emailAddressLineEdit"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>IMAP host:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="imapServerUrlLineEdit"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>SMTP user name:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLineEdit" name="smtpUserNameLineEdit">
         <property name="placeholderText">
          <string>Mostly same as the email address</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="yourNameLineEdit"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>IMAP port:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="imapServerPortSpinBox">
         <property name="maximum">
          <number>65535</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="imapUserNameLineEdit">
         <property name="inputMask">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Mostly same as the email address</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QCheckBox" name="imapEncryptedConnectionCheckbox">
         <property name="text">
          <string>Use encrypted connection</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0" colspan="2">
        <widget class="QCheckBox" name="smtpEncryptedConnectionCheckBox">
         <property name="text">
          <string>Use encrypted connection</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>yourNameLineEdit</tabstop>
  <tabstop>emailAddressLineEdit</tabstop>
  <tabstop>imapUserNameLineEdit</tabstop>
  <tabstop>imapPasswordLineEdit</tabstop>
  <tabstop>imapServerUrlLineEdit</tabstop>
  <tabstop>imapServerPortSpinBox</tabstop>
  <tabstop>imapEncryptedConnectionCheckbox</tabstop>
  <tabstop>smtpUserNameLineEdit</tabstop>
  <tabstop>smtpPasswordLineEdit</tabstop>
  <tabstop>smtpServerUrlLineEdit</tabstop>
  <tabstop>smtpServerPortSpinbox</tabstop>
  <tabstop>smtpEncryptedConnectionCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
